include/deployer.bash

```diff
@@ -499,6 +499,7 @@ main() {
         cmd-export util-token
         cmd-export util-local-dev
         cmd-export util-cleanup
+        cmd-export util-cbd-smartsense
```

Comment:
This way the command will be named: `cbd util cbd-smartsense`.
Maybe `cbd util smartsense` is more simple.